acceptance/fips/test/integration/fips/serverspec/Gemfile.lock

```diff
@@ -0,0 +1,13 @@
+GEM
```

Comment:
@tyler-ball I just went through every Gemfile I could find and locked it, on the principle that tests need to be locked just as much as the code.